pallets/ddc-clusters/src/weights.rs

```diff
@@ -31,108 +31,168 @@ pub trait WeightInfo {
 	fn add_node() -> Weight;
 	fn remove_node() -> Weight;
 	fn set_cluster_params() -> Weight;
-	fn set_cluster_gov_params() -> Weight;
+	fn validate_node() -> Weight;
 }
 
 /// Weights for pallet_ddc_clusters using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: DdcClusters Clusters (r:1 w:1)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:0 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:0 w:1)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	fn create_cluster() -> Weight {
-		Weight::from_parts(15_000_000_u64, 0)
+		Weight::from_parts(350_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine